src/firefly/js/ui/ThemeSetup.js

```diff
@@ -1,61 +1,46 @@
 import {extendTheme} from '@mui/joy';
 
+
 export function getTheme() {
     return extendTheme({
         components: {
             JoyButton: {
                 defaultProps: {
-                    size:'sm' ,
                     variant:'soft' ,
                     color:'neutral'
                 }
             },
             JoyInput: {
-                defaultProps: { size:'sm' }
+                styleOverrides: {
+                    root: {
+                        minHeight: '1.75rem',
+                    },
+                },
             },
             JoyIconButton: {
                 defaultProps: {
-                    size:'sm' ,
                     variant:'plain',
                     color:'neutral',
                 }
             },
-            JoyFormControl: {
-                defaultProps: { size:'sm' }
-            },
             JoyFormLabel: {
-                // styleOverrides: {
-                //     root: ({theme}) => ({lineHeight:'50px'})
+                // defaultProps: {
+                //     sx : {
+                //         '--FormLabel-lineHeight' : 1.1
+                //     }
                 // }
-                defaultProps: {
-                    size:'sm',
-                    sx : {
-                        '--FormLabel-lineHeight' : 1.1
-                    }
```

Comment:
I might need to revisit this one later. The labels don't seem to sit close enough to the input component. However, we can see how the small font plays our first.